### PR TITLE
Fix cd plugin/panel and transcoder

### LINKED ISSUE #320: CD ripping / "Importing": Unable to set format
transcoder.py specifies a `FORMATS` array of encoder information. But on the "Audio Disc" panel there is no way to set any format. I couldn't find one in preferences either.

Furthermore, the "steps" metadata is outdated, at least for FLAC.

### PR DESCRIPTION
**Pull request by genodeftest**
Fixes #320 and a few other issues related to the CD plugin/panel, making it work. Still room for improvement, but enough to ship in Exaile 4.0 pre-release for now in my opinion.

flac quality levels: There is a related GStreamer bug: https://bugzilla.gnome.org/show_bug.cgi?id=776413
tl;dr: GStreamer says, flac has quality levels 0…9 but it really has just level 0…8 so setting level 9 will always fail.

### CODE REVIEW

File: plugins/cd/cdprefs.py [hunk 1]
```diff
@@ -25,14 +25,12 @@
...
-from gi.repository import GObject
-from gi.repository import Gtk
 import os
-import warnings 
 
-from xl import event, settings, transcoder
+from xl import settings, transcoder
 from xl.nls import gettext as _
 from xlgui.preferences import widgets
+from __builtin__ import str
```
Comment (virtuald) [inline plugins/cd/cdprefs.py]: Why is this here?

Comment (genodeftest) [inline plugins/cd/cdprefs.py]: Probably a bug in PyDev.

File: plugins/cd/cdprefs.py [hunk 2]
```diff
@@ -30,7 +30,6 @@
 from xl import settings, transcoder
 from xl.nls import gettext as _
 from xlgui.preferences import widgets
-from __builtin__ import str
```
Comment (genodeftest) [inline plugins/cd/cdprefs.py]: Or should stash this change into the original pull request which introduced this change?

Comment (virtuald) [inline plugins/cd/cdprefs.py]: I personally like rebasing such changes, but I'll leave it up to you.

**Comment by genodeftest** [comment]
More compact version now.
